src/ocl/convolutionocl.cpp

```diff
@@ -4187,6 +4187,8 @@ void ConvolutionDescriptor::ConvolutionBackwardWeights(Handle& handle,
         break;
         case miopenConvolutionBwdWeightsAlgoImplicitGEMM:
         {
+            float elapsed = 0.0;
+            handle.ResetKernelTime();
```

Comment:
If you are doing the accumulation yourself, then this is redundant

Reply:
Thanks. Done.